spec/forms/password_form_spec.rb

```diff
@@ -13,7 +13,6 @@
   end
 
   it_behaves_like 'password validation'
-  it_behaves_like 'strong password', 'PasswordForm'
```

Comment:
This is what triggered the shared examples to be tested. I think we'll need to bring this back if we want to enforce those expectations?

Same note applies to files below `spec/forms/reset_password_form_spec.rb` and `spec/forms/update_user_password_form_spec.rb`.

Reply:
I created a separate method that checks for email address substring and Login.gov since that was previously included with the zxcvbn library validation. Please especially look at my syntax on form_password_validator.rb.